chtf/chtf.fish

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2017 Alex Kulbii
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# 'Software'), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+# CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+set CHTF_VERSION (string split "=" (head -1 (dirname (status --current-filename))/version.env))[2]
```

Comment:
This looks a bit horrible, but as `bash` and `fish` use different format for setting variable value - that seems the easiest way to get value from `bash`-formatted file.

Reply:
Would it be easier then if the file would only contain the version number? Then in bash it could be read with something like

```bash
CHTF_VERSION="$(< $(dirname $BASH_SOURCE)/VERSION)"
```

Reply:
Oh, we want to support other posix shells, so better to replace the `<` with `cat` for compatibility